### DISCUSSION TITLE
What is the I2C address of bno055

### DISCUSSION THREAD

**Discussion by ZJyche** [iNavFlight/inav#7782] [Q&A]
I have a GY-bno055. Its I2C address is 0x29. Can I use it for imu2 of inav?

**Comment by SebKister** [comment]
Yes you are good to go, the address is set at 0x29:
 BUSDEV_REGISTER_I2C(busdev_bno055,      DEVHW_BNO055,       BNO055_I2C_BUS,     0x29,               NONE,           DEVFLAGS_NONE, 0);
